src/core/server/integration_tests/ui_settings/doc_exists.ts

```diff
@@ -201,6 +201,36 @@ export const docExistsSuite = (savedObjectsIndex: string) => () => {
     });
   });
 
+  describe('validate route', () => {
+    it('returns a 200 and a validation error message if value is invalid', async () => {
```

Comment:
Usually we return `400` for invalid request payloads. What's the reasoning behind using `200` here?

Reply:
I was thinking that, since the purpose of this route is to validate a value and return a validation error message if one is thrown, we can accept any value and return `200` and an empty body if the value is valid, or `200` and a response body containing the validation error message if the value is invalid. WDYT?